projects/BFDR/BirthRecord_submissionProperties.cs

```diff
@@ -2355,6 +2355,208 @@ public string FinalRouteAndMethodOfDeliveryHelper
+        /// <para>// Getter:</para>
+        /// <para>Console.WriteLine($"Mother transferred: {ExampleBirthRecord.LaborTrialAttempted}");</para>
+        /// </example>
+        [Property("LaborTrialAttempted", Property.Types.Bool, "LaborTrialAttempted", "LaborTrialAttempted", false, BFDR.IGURL.ObservationLaborTrialAttempted, true, 288)]
+        [FHIRPath(fhirType: FHIRPath.FhirType.Observation, categoryCode: "73760-1", section: MEDICAL_INFORMATION_SECTION)]
+        public bool? LaborTrialAttempted
+        {
+            get
+            {
+                Observation obs = GetObservation("73760-1");
+                if (obs != null)
+                {
+                    bool? infantLiving = ((FhirBoolean)obs.Value).Value;
+                    return infantLiving;
+                }
+                return null;
+            }
+            set
+            {
+                if (value == null)
+                {
+                    return;
+                }
+                Observation obs = GetOrCreateObservation("73760-1", CodeSystems.LOINC, "Labor Trial Attempted", BFDR.ProfileURL.ObservationLaborTrialAttempted, MEDICAL_INFORMATION_SECTION);
+                obs.Value = new FhirBoolean(value);
+            }
+        }
+
+        /// <summary>NumberOfPreviousCesareans.</summary>
+        /// <value>NumberOfPreviousCesareans</value>
+        /// <example>
+        /// <para>// Setter:</para>
+        /// <para>ExampleBirthRecord.NumberOfPreviousCesareans = 1;</para>
+        /// <para>// Getter:</para>
+        /// <para>Console.WriteLine($"NumberOfPreviousCesareans: {ExampleBirthRecord.NumberOfPreviousCesareans}");</para>
+        /// </example>
+        [Property("Number Of Previous Cesareans", Property.Types.Int32, "Number Of Previous Cesareans", "Number Of Previous Cesareans.", true, IGURL.ObservationNumberPreviousCesareans, true, 14)]
+        [FHIRPath("Bundle.entry.resource.where($this is Observation).where(code.coding.code='68497-7')", "")]
+        public int? NumberOfPreviousCesareans
+        {
+            get
+            {
+                Observation obs = GetObservation("68497-7");
+                if (obs != null)
+                {
+                    if (obs != null && obs.Value != null && obs.Value as Hl7.Fhir.Model.Integer != null)
+                    {
+                        return (obs.Value as Hl7.Fhir.Model.Integer).Value;
+                    }
+                }
+                return null;
+            }
+            set
+            {
+                if (value == null)
+                {
+                    return;
+                }
+                Observation obs = GetObservation("68497-7");
+                if (obs == null)
+                {
+                    obs = GetOrCreateObservation("68497-7", CodeSystems.LOINC, BFDR.ProfileURL.ObservationNumberPrenatalVisits, MEDICAL_INFORMATION_SECTION, Mother.Id);
+                    obs.Value = new Hl7.Fhir.Model.Integer();
```

Comment:
I don't think you need this line since a new Integer is being created on the next line anyway?

Reply:
removed